src/components/IconButton/IconButton.visual.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { IconButton, Skins } from './';
+import { ReactComponent as ShareIcon } from '../../assets/icons/Share.svg';
+import { VisualContainerElement } from '../../../test/visual/VisualContainerElement';
+
+class IconButtonVisual extends React.Component<any> {
+  static defaultProps = {
+    mobile: false,
+  };
+
+  render() {
+    return (
+      <VisualContainerElement>
+        <IconButton
+          icon={<ShareIcon />}
+          data-hook={'storybook-e2e-IconButton'}
+          {...this.props}
+        />
+        <IconButton
+          icon={<ShareIcon />}
+          data-hook={'storybook-e2e-IconButton'}
+          skin={Skins.Full}
+          {...this.props}
+          disabled
```

Comment:
instead of having the `disabled` property here, lets add it to the array of tests below:
```js
const tests = [
  {
    describe: 'basic',
    its: [
      {
        it: 'default',
        props: {},
      },
      {
        it: 'default',
        props: {
          disabled: true,
        },
      },
    ],
  },
];
```

Reply:
and the render function will be:
```
<VisualContainerElement>
        <IconButton
          icon={<ShareIcon />}
          {...this.props}
        />
        <IconButton
          icon={<ShareIcon />}
          skin={Skins.Full}
          {...this.props}
        />
      </VisualContainerElement>
    );
```
(we can also remove the data hooks, we'll remove it from generator)